components/button/index.css

```diff
@@ -167,10 +136,15 @@ a.spectrum-Button {
   @inherit: %spectrum-ButtonLabel;
   padding-block-start: var(--mod-button-padding-label-top, var(--spectrum-button-padding-label-top));
   padding-block-end: var(--mod-button-padding-label-bottom, var(--spectrum-button-padding-label-bottom));
+  white-space: nowrap;
+  line-height: calc(
```

Comment:
Prevents the label from being "taller" than the button, which I feel normalized the vertical positioning of the label, which moved _up_ in the Core Tokens refactor, but it moves up/down a lot and I'm never sure what design really wants here.